DomainParser/DomainParserTests/DomainParserTests.swift

```diff
@@ -130,5 +130,16 @@ class DomainParserTests: XCTestCase {
         guard let host = host else { return }
         XCTAssertEqual(domainParser.parse(host: host.lowercased())?.domain, expectedDomain, file: file, line: line)
     }
+
+
+    func testTLDWithNoDomain() {
+        XCTAssertEqual(domainParser.parse(host: "com"), ParsedHost(publicSuffix: "com", domain: nil))
```

Comment:
Before the changes, this assert would fail but the rest would pass.
Now all the asserts in this test pass.